test/var_compatibility_test/test_host_mod.F90

```diff
@@ -37,6 +37,7 @@ subroutine init_data()
        phys_state%effri = 5.0E-5 ! 50 microns, in meter
        phys_state%nci = 80
     endif
+    phys_state%tke = 10.0 !J kg-1
```

Comment:
how are we confirming that the equivalent units aren't impacting this value?

I think `effr_calc_run` should modify turbulent_kinetic_energy and then we should confirm that the result is right in `compare_data` below.

Reply:
@peverwhee @cacraigucar 
I added a PR to @climbfuji  branch in #656